.github/workflows/release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
-      - uses: sigstore/cosign-installer@48866aa521d8bf870604709cd43ec2f602d03ff2 # v2.4.1
```

Comment:
What do we want to do here?

- let this stay out of date?
- manually suggest and commit an edit every time dependabot updates this? (fails DCO)
- remove the comment, losing valuable information?
- use something besides dependabot to update these, that updates the comment?

cc @cpanato @wlynch

Reply:
i usually push a commit to update the version

Reply:
just updated.